example/src/main/java/org/wordpress/android/fluxc/example/ui/products/WooProductsFragment.kt

```diff
@@ -371,6 +377,26 @@ class WooProductsFragment : Fragment() {
             }
         }
 
+        fetch_product_addons.setOnClickListener {
+            selectedSite?.let { site ->
+                showSingleLineDialog(
+                        activity,
+                        "Enter the remoteProductId of product to fetch the addons:"
+                ) { editText ->
+                    pendingFetchSingleProductRemoteId = editText.text.toString().toLongOrNull()
+                    pendingFetchSingleProductRemoteId?.let { id ->
+                        prependToLog("Submitting request to fetch product by remoteProductID $id")
+                        coroutineScope.launch {
+                            wcProductStore.fetchProductListSynced(site, listOf(id))
+                                    .takeUnless { it.isNullOrEmpty() }
+                                    ?.first()?.addons
+                                    ?.logAddons()
```

Comment:
I think here we also have to adjust the code: 
```suggestion
                                    ?.first()?.addons
                                    .logAddons()
```

Otherwise, when `addons` is `null`, `logAddons()` method won't be executed. I've tested it after removing `?` and now it seems to work 🙂:

![image](https://user-images.githubusercontent.com/5845095/126976021-2443fb4c-b9e7-4b3c-9664-b3df52a657ae.png)

Reply:
This was a lack of attention of mine, sorry, it's fixed here: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2066/commits/096b4455d600ace1465357ab5e896cb7505a9afd